sample.net.cloud.google_iot_mqtt test is failing for frdm_k64f

The `samples/net/cloud/google_iot_mqtt/sample.net.cloud.google_iot_mqtt` test fails to build for the `frdm_k64f` platform. Currently reproducible on master.

```
$ ./scripts/sanitycheck -p frdm_k64f --testcase-root samples/net/cloud/google_iot_mqtt/
...
Zephyr version: 2.1.99
-- Found PythonInterp: /usr/bin/python3 (found suitable version "3.7.5", minimum required is "3.4") 
-- Selected BOARD frdm_k64f
-- Found west: /home/carlo/.local/bin/west (found suitable version "0.6.2", minimum required is "0.6.0")
-- Loading /home/carlo/work/proj/zephyr/zephyrproject/zephyr/boards/arm/frdm_k64f/frdm_k64f.dts as base
Devicetree configuration written to /home/carlo/work/proj/zephyr/zephyrproject/zephyr/sanity-out/frdm_k64f/samples/net/cloud/google_iot_mqtt/sample.net.cloud.google_iot_mqtt/zephyr/include/generated/generated_dts_board.conf
Parsing Kconfig tree in /home/carlo/work/proj/zephyr/zephyrproject/zephyr/samples/net/cloud/google_iot_mqtt/Kconfig
Loaded configuration '/home/carlo/work/proj/zephyr/zephyrproject/zephyr/boards/arm/frdm_k64f/frdm_k64f_defconfig'
Merged configuration '/home/carlo/work/proj/zephyr/zephyrproject/zephyr/samples/net/cloud/google_iot_mqtt/prj.conf'
Configuration saved to '/home/carlo/work/proj/zephyr/zephyrproject/zephyr/sanity-out/frdm_k64f/samples/net/cloud/google_iot_mqtt/sample.net.cloud.google_iot_mqtt/zephyr/.config'
-- The C compiler identification is GNU 9.2.0
-- The CXX compiler identification is GNU 9.2.0
-- The ASM compiler identification is GNU
-- Found assembler: /home/carlo/work/proj/zephyr/zephyr-sdk/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc
-- Cache files will be written to: /home/carlo/.cache/zephyr
CMake Error at CMakeLists.txt:9 (message):
  !!!!!! Generate key file before continuing.  See README !!!!!


-- Configuring incomplete, errors occurred!
See also "/home/carlo/work/proj/zephyr/zephyrproject/zephyr/sanity-out/frdm_k64f/samples/net/cloud/google_iot_mqtt/sample.net.cloud.google_iot_mqtt/CMakeFiles/CMakeOutput.log".
See also "/home/carlo/work/proj/zephyr/zephyrproject/zephyr/sanity-out/frdm_k64f/samples/net/cloud/google_iot_mqtt/sample.net.cloud.google_iot_mqtt/CMakeFiles/CMakeError.log".
```

This has been caught by shippable on #21326.

--- a/samples/net/cloud/google_iot_mqtt/sample.yaml
+++ b/samples/net/cloud/google_iot_mqtt/sample.yaml
@@ -1,8 +1,6 @@
 sample:
   description: MQTT sample app to Google IoT cloud
   name: google_iot_mqtt
-tests:
-  sample.net.cloud.google_iot_mqtt:
-    harness: net
-    platform_whitelist: frdm_k64f
-    tags: net mqtt cloud
+common:
+  tags: net mqtt cloud
+  harness: net

